packages/jest-cli/src/reporters/JestReporterVerbose.js

```diff
@@ -85,8 +69,31 @@ class VerboseLogger {
 
   _logLine(str?: string, indentLevel?: number) {
     const indentation = '  '.repeat(indentLevel || 0);
-    this._process.stdout.write(indentation + (str || '') + '\n');
+    process.stderr.write(indentation + (str || '') + '\n');
+  }
+
+  static groupTestsBySuites(testResults: Array<AssertionResult>) {
```

Comment:
Can we leave statics right after the constructor?

(We should add a lint rule for this)

It should be something like:

* constructor
* statics
* public methods
* private methods